pkg/windows/windows.go

```diff
@@ -234,6 +234,8 @@ type Windows interface {
 	ConfigureWICD(string, string, *Authentication) error
 	// Deconfigure removes all files and networks created by WMCO and runs the WICD cleanup command.
 	Deconfigure(string, string, *Authentication) error
+	// DeconfigureWICD ensures the WICD service running on the Windows instance is removed
+	DeconfigureWICD() error
 	// RunWICDCleanup runs the WICD cleanup command that ensures all services configured by WICD are stopped.
 	RunWICDCleanup(string, string) error
```

Comment:
I don't think we need two separate methods for this. Instead you can just ensure the service is stopped within  RunWICDCleanup.